config/brakeman.ignore

```diff
@@ -0,0 +1,25 @@
+{
+  "ignored_warnings": [
+    {
+      "warning_type": "Redirect",
+      "warning_code": 18,
+      "fingerprint": "1f5e56ec432800fd1896435e9cd488f3c75047cdb1855add003acf99052a04f3",
+      "message": "Possible unprotected redirect",
+      "file": "app/controllers/authentications_controller.rb",
+      "line": 4,
+      "link": "http://brakemanscanner.org/docs/warning_types/redirect/",
+      "code": "redirect_to(Authenticator.new(request.env[\"omniauth.auth\"], session).perform)",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "AuthenticationsController",
+        "method": "create"
+      },
+      "user_input": "Authenticator.new(request.env[\"omniauth.auth\"], session).perform",
+      "confidence": "High",
+      "note": "Fixed here: https://github.com/18F/micropurchase/commit/799d74393c394e016fba5eab9df1eb638a7d2b06"
+    }
```

Comment:
Yes it is. The change doesn't silence the warning. The note in this file
points to the commit which protects against the (already low low risk)
warning.

On Thursday, November 19, 2015, Aidan Feldman <notifications@github.com>
wrote:

> In config/brakeman.ignore
> <https://github.com/18F/micropurchase/pull/90#discussion_r45389666>:
>
> > +      "fingerprint": "1f5e56ec432800fd1896435e9cd488f3c75047cdb1855add003acf99052a04f3",
> > +      "message": "Possible unprotected redirect",
> > +      "file": "app/controllers/authentications_controller.rb",
> > +      "line": 4,
> > +      "link": "http://brakemanscanner.org/docs/warning_types/redirect/",
> > +      "code": "redirect_to(Authenticator.new(request.env[\"omniauth.auth\"], session).perform)",
> > +      "render_path": null,
> > +      "location": {
> > +        "type": "method",
> > +        "class": "AuthenticationsController",
> > +        "method": "create"
> > +      },
> > +      "user_input": "Authenticator.new(request.env[\"omniauth.auth\"], session).perform",
> > +      "confidence": "High",
> > +      "note": "Fixed here: https://github.com/18F/micropurchase/commit/799d74393c394e016fba5eab9df1eb638a7d2b06"
> > +    }
>
> Is this necessary, with the change to the controller?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/micropurchase/pull/90/files#r45389666>.
>